### PR TITLE
Cleanup test-package.yml

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: .github/workflows/test-package.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   # Check code formatting and static analysis on a single OS (linux)
-  # against Dart dev.
+  # against Dart dev & stable.
```
Comment (athomas) [inline .github/workflows/test-package.yml]: You didn't actually add stable to the matrix.

File: .github/workflows/test-package.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
         sdk: [dev]
     steps:
       - uses: actions/checkout@v2
-      - uses: dart-lang/setup-dart@v0.1
+      - uses: dart-lang/setup-dart@v1.0
         with:
           channel: ${{ matrix.sdk }}
```
Comment (athomas) [inline .github/workflows/test-package.yml]: `sdk: ${{}}`

File: .github/workflows/test-package.yml [hunk 3]
```diff
@@ -37,20 +37,19 @@ jobs:
...
-        # Add macos-latest and/or windows-latest if relevant for this package.
         os: [ubuntu-latest]
-        sdk: [dev]
+        sdk: [dev, stable]
     steps:
       - uses: actions/checkout@v2
-      - uses: dart-lang/setup-dart@v0.1
+      - uses: dart-lang/setup-dart@v1.0
         with:
           channel: ${{ matrix.sdk }}
```
Comment (athomas) [inline .github/workflows/test-package.yml]: `sdk: ${{}}`

File: .github/workflows/test-package.yml [hunk 4]
```diff
@@ -37,20 +37,19 @@ jobs:
...
   test:
     needs: analyze
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        # Add macos-latest and/or windows-latest if relevant for this package.
         os: [ubuntu-latest]
-        sdk: [dev]
+        sdk: [dev, stable]
```
Comment (athomas) [inline .github/workflows/test-package.yml]: Consider `2.12.0` instead of stable (the lower bound in the pubspec.yaml).

**Review by athomas — CHANGES_REQUESTED** [review]

**Comment by mit-mit** [comment]
Thanks, PTAL

**Review by athomas — APPROVED** [review]

**Pull request merged**